plugins/snapshots/windows/cimfs.go

```diff
@@ -327,27 +319,37 @@ func (s *cimFSSnapshotter) mounts(sn storage.Snapshot, key string) []mount.Mount
 
 	source := s.getSnapshotDir(sn.ID)
 	parentLayerPaths := s.parentIDsToParentPaths(sn.ParentIDs)
+	layerCimPaths := s.parentIDsToCimPaths(sn.ParentIDs)
 
 	mountType := "CimFS"
 
 	// error is not checked here, as a string array will never fail to Marshal
 	parentLayersJSON, _ := json.Marshal(parentLayerPaths)
 	parentLayersOption := mount.ParentLayerPathsFlag + string(parentLayersJSON)
+	parentCimLayersJSON, _ := json.Marshal(layerCimPaths)
+	parentCimLayersOption := mount.ParentLayerCimPathsFlag + string(parentCimLayersJSON)
 
 	options := []string{
 		roFlag,
 	}
 	if len(sn.ParentIDs) != 0 {
 		options = append(options, parentLayersOption)
+		options = append(options, parentCimLayersOption)
+	}
+	// if this is an image layer being extracted include a cim path in which the layer
+	// will be extracted.
+	if strings.Contains(key, snapshots.UnpackKeyPrefix) {
```

Comment:
It would be nice to avoid keying more stuff off of `UnpackKeyPrefix`. Is there any way we can avoid this?

Reply:
Actually, there are two other places (Commit & Mounts) where we still need to use the UnpackKeyPrefix. I am trying to figure out if we can easily change it there too. But if not, would you be okay with making this change in a separate PR?

Reply:
What about just always including the `cimpath` option in the mounts? Then we just ignore that option when it's not needed. This also has the (potentially desirable) side effect where mounts don't change for a snapshot when it's committed.

Reply:
I have updated the code to always include the `cimpath` option in the mounts. Please take another look.